src/one_dragon_qt/widgets/setting_card/multi_push_setting_card.py

```
@@ -55,7 +55,7 @@ def __init__(self,
                  icon: Union[str, QIcon, FluentIconBase], title: str, content: Optional[str]=None,
                  icon_size: IconSize = IconSize(16, 16),
                  margins: Margins = Margins(16, 16, 0, 16),
-                 line_list: List[List[QAbstractButton]] = None,
+                 line_list: List[List[QWidget]] = None,
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)